tests/framework/utils_drive.py

```diff
@@ -63,3 +85,17 @@ def spawn_vhost_user_backend(vm, host_mem_path, socket_path, readonly=False):
         os.chown(socket_path, uid, gid)
 
     return proc
+
+
+def resize_vhost_user_drive(new_size, backend="crosvm"):
+    """
+    Resize vhost-user-blk drive and send config change notification.
+
+    New size is in MB.
+    """
+
+    if backend != "crosvm":
+        assert False, f"backend `{backend}` does not support resizing"
+    assert os.path.exists(CROSVM_CTR_SOCKET), "crosvm backend must be spawned first"
+
+    utils.run_cmd(f"crosvm disk resize 0 {new_size * 1024 * 1024} {CROSVM_CTR_SOCKET}")
```

Comment:
This having a `backend` parameter only for it to check that the parameter has a specific value feels a bit weird. Isnt the "path exists" check enough here?

Reply:
done